src/features/admin/views/tags/create.tsx

```diff
@@ -0,0 +1,72 @@
+import { useForm } from "@/hooks/useForm";
+import { api } from "@/utils/api";
+import { Button, Heading, Input, useToast, VStack } from "@chakra-ui/react";
+import React from "react";
+import { z } from "zod";
+import { FormField } from "../../components/FormField";
+import { Layout } from "../../components/Layout";
+
+const schema = z.object({
+  text: z
+    .string()
+    .min(1, "Tag nie może być pusty")
+    .max(100, "Tag może mieć maksymalnie 100 znaków"),
```

Comment:
Tutaj mnie to zastanawia, gdyż tag powinien być krótki, pamiętliwy i łatwy do zapamiętania, więc na pewno zmieniszyć limit ze 100 na 20-30, np. tag "programowanie" ma 13, jeżeli byśmy chcieli doprecyzować to prędzej lepiej zrobić osobnego taga niż robić dużego taga np. "programowanie obiektowe", te tagi otrzymane od "klienta" są nieodpowiednie, gdyż są zbyt długie i zbyt precyzyjne